WalletWasabi.Fluent/Views/Wallets/Home/History/Details/TransactionDetailsView.axaml

```diff
@@ -25,14 +35,28 @@
       <!-- Amount -->
       <c:PreviewItem Icon="{StaticResource btc_logo}"
                      Label="{Binding AmountText}"
-                     TextValue="{Binding Amount}">
+                     CopyableContent="{Binding Amount}">
         <c:PrivacyContentControl>
-          <TextBlock Text="{Binding Amount, StringFormat={}{0} BTC, Mode=OneWay}" Classes="monoSpaced" />
+          <TextBlock Text="{Binding Amount, Converter={x:Static converters:MoneyConverters.ToBtc}}" Classes="monoSpaced" />
         </c:PrivacyContentControl>
       </c:PreviewItem>
 
       <Separator />
 
+      <!-- Fee -->
+      <StackPanel IsVisible="{Binding IsFeeVisible}">
+        
+        <c:PreviewItem Icon="{StaticResource btc_logo}"
```

Comment:
use the same logo as on the transaction preview screen.

Reply:
Done.